src/extractor/parserVue/tokenMergers/hyphenPropsMerger.ts

```diff
@@ -0,0 +1,30 @@
+import { MachineType } from '../../parser/mergerMachine.js';
+import { VueMappedTokenType } from '../ParserVue.js';
+
+export const enum S {
+  Idle,
+  ExpectT,
+}
+
+// <T key-name="my_key" /> -> <T keyName="key" />
```

Comment:
```suggestion
// <T key-name="my_key" /> -> <T keyName="my_key" />
```

Just to make it less confusing that the key value gets not modified, if I understand it correctly.